WebService/appsettings.ini

```diff
@@ -153,6 +153,14 @@ azure_resource_group = "${?PCS_RESOURCE_GROUP}"
 # The value is used to create a URL taking to the IoT Hub metrics in the Azure portal.
 azure_iothub_name = "${?PCS_IOHUB_NAME}"
 
+# Location where azure resource group is deployed, e.g "East US".
+# The value is used to update auto scale settings on vmss on simulation start and stop.
+azure_resource_group_location = "${?PCS_RESOURCE_GROUP_LOCATION}"
+
+# Name of VMSS resource. E.g "test_vmss"
+# The value is used to update auto scale settings on vmss on simulation start and stop.
+azure_vmss_name = "${?PCS_VMSS_NAME}"
+
```

Comment:
why are these two new env vars optional?

Reply:
these are required for vmss scale requests. Added checks in config.

Reply:
if they are required you should remove the `?`

Reply:
Keeping these optional to make local development easier.